### ISSUE TITLE
cosmos is hardcoding node_env to 'development'

### ISSUE THREAD

**Issue by scottdickerson** [react-cosmos/react-cosmos#626]
### What's up?
When starting cosmos from the command line it's hardcoding the node_env to 'development'.  https://github.com/react-cosmos/react-cosmos/blob/63e6c0574ceb12e13a9882e7f00fa5cf4d13262a/packages/react-cosmos/bin/cosmos.js#L5  This is fine by default, but I need to override it to pass a specific node_env so that I can instrument my classes for code coverage.

### Mkay, tell me more...

Allow a node env to be passed and if none passed default to 'development'.

**Comment by skidding** [comment]
Yes! This should be fixed.

https://github.com/react-cosmos/react-cosmos/blob/0aea3ff15cb0321c0ffbd319168b2d564ab07f75/packages/react-cosmos/bin/cosmos.js#L4-L5

and 

https://github.com/react-cosmos/react-cosmos/blob/0aea3ff15cb0321c0ffbd319168b2d564ab07f75/packages/react-cosmos/bin/cosmos-export.js#L4-L5

Should be

```diff
-process.env.BABEL_ENV = 'development';
+process.env.BABEL_ENV = process.env.BABEL_ENV || 'development';
-process.env.NODE_ENV = 'development';
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```

and 

```diff
-process.env.BABEL_ENV = 'production';
+process.env.BABEL_ENV = process.env.BABEL_ENV || 'production';
-process.env.NODE_ENV = 'production';
+process.env.NODE_ENV = process.env.NODE_ENV || 'production';
```

@scottdickerson Would you be willing to submit a patch for this? I'd be happy to review and release it.

**Comment by scottdickerson** [comment]
absolutely @skidding PR https://github.com/react-cosmos/react-cosmos/pull/630 put in to fix this, thanks!

**Comment by skidding** [comment]
Thanks @scottdickerson! 

Published under `react-cosmos@4.0.0-beta.4`, check it out. 

cc @NiGhTTraX I think you needed this as well.
